editor/editor_interface.h

```diff
@@ -110,6 +123,12 @@ class EditorInterface : public Object {
 	String get_current_feature_profile() const;
 	void set_current_feature_profile(const String &p_profile_name);
 
+	// Editor dialogs.
+
+	void popup_dialog_scene_tree(const Callable &p_callback, const TypedArray<StringName> &p_valid_types = TypedArray<StringName>());
+	// Must use Vector<int> because exposing Vector<Variant::Type> is not supported.
+	void popup_dialog_property_selector(Object *p_object, const Callable &p_callback, const PackedInt32Array &p_type_filter = PackedInt32Array());
```

Comment:
`popup_dialog_*` prefixes should be reserved to generic methods that help pop up dialogs. In this case we shouldn't mix them up. I also think that methods can be named closer to their purpose (as described in the docs).

So I'd call them this way:

```suggestion
	void popup_node_selector(const Callable &p_callback, const TypedArray<StringName> &p_valid_types = TypedArray<StringName>());
	// Must use Vector<int> because exposing Vector<Variant::Type> is not supported.
	void popup_property_selector(Object *p_object, const Callable &p_callback, const PackedInt32Array &p_type_filter = PackedInt32Array());
```

Reply:
Great suggestion! I like the names you suggested. I don't think there are any other contexts where one would select a node besides inside a scene.